.github/ISSUE_TEMPLATE/discussion_issue.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Discussion Issue Template
+about: For discussing high-level decisions and strategy
+title: '[Discussion]'
+labels: 'discussion'
```

Comment:
I think "enhancement" or "question" are going to be more common than "discussion". Does anyone have an opinion regarding the default label here?

Reply:
I think @KirstieJane expressed concern about `Enhancement` as a label, but I'm also not a big fan of `Discussion`. I think questions are something we'd port to NeuroStars, ideally.

One idea: Is there a reason we need two issue templates instead of one ? Could we just have a single issue template explaining the amount of detail that we need and that more general usage-related questions should go to NeuroStars ?

Reply:
I agree that usage-related questions should go to NeuroStars, but we should still differentiate bug reports from feature requests (which we can label as something other than `enhancement` if there are problems with that). That said, I imagine that feature requests will probably come from people with more familiarity with the repository, while bug reports could come from a wider range of people. If need be, I think using the bug report template as the only one should be fine.

Reply:
Sorry for being unclear ! I mean that we could create just one issue template. It would have text at the top mentioning NeuroStars, and then could encourage cross-linking issues, having a 1-2 sentence summary of the issue, any specific detail necessary to recreate the issue or appropriate context to consider, and then suggested next steps. That would be broad enough to cover both use cases, I think !

Reply:
Okay, I'll work on this. Do you want me to close and reopen this, or just add more commits?

Reply:
Please go ahead and add more commits, here !